pyiron_atomistics/lammps/control.py

```diff
@@ -231,6 +231,9 @@ def calc_minimize(
         # defaults stay consistent!
 
         max_evaluations = 100 * max_iter
+        if n_print > max_iter:
+            warnings.warn('n_print larger than max_iter, adjusting to n_print=max_iter')
```

Comment:
I would `from pyiron_base import state` and then 
```suggestion
            state.logger.warning('n_print larger than max_iter, adjusting to n_print=max_iter')
```